lib/ansible/module_utils/basic.py

```diff
@@ -53,9 +53,7 @@
 except ImportError:
     HAS_SYSLOG = False
 
-# deprecated: description='types.EllipsisType is available in Python 3.10+' python_version='3.9'
-if t.TYPE_CHECKING:
-    from builtins import ellipsis
+_UNSET = t.cast(t.Any, ...)
```

Comment:
Can we use object instead of an arbitrarily wide type?
```suggestion
_UNSET = object()
```

Reply:
```suggestion
class Sentinel(enum.Enum):
  sentinel = 0

_UNSET = _TypedSentinel.sentinel
```

Reply:
We were originally using just `...` where `_UNSET` is now used. In the places where we made the switch I've gone back and updated those to use `object()` instead of `...` for `_UNSET`.

The reason why we're using `t.Any` for these is that we want the sentinel to be accepted by `mypy`, but not to show up in the type annotation as one of the acceptable types for the argument. It will, of course, show up as the default value, but that's unavoidable without switching over to `**kwargs`.

These `_UNSET` arguments are primarily for detecting usage of arguments that have been deprecated, where we needed an easy way to detect they had been passed so a deprecation warning could be issued.